go-controller/pkg/node/node.go

```diff
@@ -170,10 +170,9 @@ func (n *OvnNode) Start(wg *sync.WaitGroup) error {
 		klog.Errorf("Setting klog \"loglevel\" to 5 failed, err: %v", err)
 	}
 
-	for _, auth := range []config.OvnAuthConfig{config.OvnNorth, config.OvnSouth} {
-		if err := auth.SetDBAuth(); err != nil {
-			return err
-		}
+	// ovn-controller communicates with Southbound DB only.
+	if err := config.OvnSouth.SetDBAuth(); err != nil {
+		return err
```

Comment:
@roytman yes we will not need `ovn-nb`. However,  I think you need to make severa otherl changes as well
- fix config_test.go to not look for `ovn-nb` external_ids anymore
- fix buildOvnAuth() function to not set `auth.externalID = "ovn-nb"` or perhaps remove `.externalID` field altogether.

We still need to call SetDBAuth() for Both NB and SB since we call ensureCACert()
```		
// Client can bootstrap the CA cert from the DB
		if err := a.ensureCACert(); err != nil {
			return err
		}
```

Reply:
@girishmg , thanks for the review. Hopefully, I fixed all comments.